com.unity.shadergraph/Editor/ShaderFoundry/Utilities/PropertyAttribute.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+
+namespace UnityEditor.ShaderFoundry
+{
+    internal enum UniformDataSource { Global, PerMaterial, PerInstance, Custom, None = -1 }
```

Comment:
The enum values here follow a different naming convention as compared to KeywordMode